compose.yml

```diff
@@ -0,0 +1,58 @@
+x-common:
+  panel:
+    &panel-environment
+    APP_URL: "https://localhost" # can be set to 'http://localhost' to un on port 80 only
```

Comment:
this should be defaulting to `http://` not `https://` it throws errors about certificates.